src/main/java/org/jenkinsci/plugins/JiraTestResultReporter/JiraTestDataPublisher.java

```diff
@@ -251,9 +264,11 @@ private boolean raiseIssues(TaskListener listener, Job project, Job job,
                              EnvVars envVars,List<CaseResult> testCaseResults) {
         boolean raised = false;
         for(CaseResult test : testCaseResults) {
-            if(test.isFailed() && TestToIssueMapping.getInstance().getTestIssueKey(job, test.getId()) == null) {
+            if(test.isFailed()) {
                 try {
-                    JiraUtils.createIssue(job, project, envVars, test);
+                    if (TestToIssueMapping.getInstance().getTestIssueKey(job, test.getId()) == null || test.isFailed()) {
+                        JiraUtils.createIssue(job, project, envVars, test);
+                    }
```

Comment:
`test.isFailed()` is always true at this point (because of line 267). So this conditional block (line 269-271) is effectively the same as
```suggestion
                    TestToIssueMapping.getInstance().getTestIssueKey(job, test.getId());
                    JiraUtils.createIssue(job, project, envVars, test);
```
But is that what you intend?

Reply:
Fixed in https://github.com/jenkinsci/JiraTestResultReporter-plugin/pull/24/commits/220429f18a2511d77815b5320e02113b6136fda4. Missed to remove that non-sense condition check from my testing